test/auth_mvp_web/controllers/github_auth_controller_test.exs

```diff
@@ -4,5 +4,9 @@ defmodule AuthMvpWeb.GithubAuthControllerTest do
   test "GET /auth/github/callback", %{conn: conn} do
     conn = get(conn, Routes.github_auth_path(conn, :index, %{code: "123", state: "http://localhost/"}))
     assert html_response(conn, 302)
+
+    # same again to exercise to the branch where person already exists:
```

Comment:
you were already so close to 100% coverage that I just added these extra lines to get to 💯 
(_hope you don't mind ..._)